core/http/server-spring/src/main/java/org/eclipse/rdf4j/http/server/repository/namespaces/NamespaceController.java

```diff
@@ -84,13 +84,10 @@ else if ("DELETE".equals(reqMethod)) {
 	private ModelAndView getExportNamespaceResult(HttpServletRequest request, String prefix)
 		throws ServerHTTPException, ClientHTTPException
 	{
-		try {
+		try (RepositoryConnection repositoryCon = RepositoryInterceptor.getRepositoryConnection(request)) {
 			String namespace = null;
```

Comment:
namespace = null is redundant now that the synchronized section has been removed